### PR TITLE
fix typos (work done by Andrea Gelmini)

### PR DESCRIPTION
**Pull request by ThomasWaldmann**
Submitted on behalf of a third-party: Andrea Gelmini @Gelma

### CODE REVIEW

**Review by Cyan4973 — APPROVED** [review]

**Comment by ThomasWaldmann** [comment]
I don't think fixing typos is copyrightable work, so guess signing a CLA is not necessary here, right?

**Comment by Cyan4973** [comment]
Unfortunately, that's not an option.

> "do not merge anything without the CLA being signed. Even typos fixes."

**Comment by ThomasWaldmann** [comment]
OK, considering it was not me who found/fixed the typos, who needs to sign? Me or @Gelma?

These changes were originally part of https://github.com/borgbackup/borg/pull/5597.

**Comment by Cyan4973** [comment]
Considering the process in place, it's whoever pushes the PR.

File: lib/compress/zstd_ldm.c [hunk 1]
```diff
@@ -620,7 +620,7 @@ size_t ZSTD_ldm_blockCompress(rawSeqStore_t* rawSeqStore,
 
     assert(rawSeqStore->pos <= rawSeqStore->size);
     assert(rawSeqStore->size <= rawSeqStore->capacity);
-    /* Loop through each sequence and apply the block compressor to the lits */
+    /* Loop through each sequence and apply the block compressor to the lists */
```
Comment (terrelln) [inline lib/compress/zstd_ldm.c]: This isn't a typo: `lits` is correct

Comment (terrelln) [reply lib/compress/zstd_ldm.c]: It could be extended from `lits` to `literals`

Comment (ThomasWaldmann) [reply lib/compress/zstd_ldm.c]: fixed, thanks for the hint.

**Comment by ThomasWaldmann** [comment]
OK, I signed the CLA, see also top post.

**Comment by Cyan4973** [comment]
Thanks @ThomasWaldmann !

**Pull request merged**